sklearn/inspection/tests/test_partial_dependence.py

```diff
@@ -502,10 +502,23 @@ def test_plot_partial_dependence_multiclass(pyplot):
     plot_partial_dependence(clf, iris.data, [0, 1],
                             target='setosa',
                             grid_resolution=grid_resolution)
-    fig = pyplot.gcf()
-    axs = fig.get_axes()
-    assert len(axs) == 2
-    assert all(ax.has_data for ax in axs)
+    fig2 = pyplot.gcf()
+    axs2 = fig2.get_axes()
+    assert len(axs2) == 2
+    assert all(ax.has_data for ax in axs2)
+
+    # check that the pd plots are the same for 0 and "setosa"
+    assert all(axs[0].lines[0]._y == axs2[0].lines[0]._y)
+    # check that the pd plots are different for another target
+    clf = GradientBoostingClassifier(n_estimators=10, random_state=1)
```

Comment:
think you can remove a few lines, namely the `clf` definition and fitting, as well as the `grid_resolution`.

Reply:
Ok, I will. I did not change those because I did not know if it had to be with some standard you use when testing.